### ISSUE TITLE
Need more semantic icons

### ISSUE THREAD

**Issue by sirbrillig** [sirbrillig/gitnews-menubar#48]
Forked from #43. 

I need to create new icons for the various status cases that can be differentiated without color:

- No notifications ("Normal"; currently all black with no dot)
- Unseen notifications ("Alert"; currently all orange with no dot)
- Seen but unread notifications ("Warn"; currently all black with orange dot)
- Offline ("Offline"; currently all black with red dot)
- Errors ("Error"; currently all red with no dot)

As discussed on the original issue, we could combine error and offline.

**Comment by sirbrillig** [comment]
cc @michaeldcain

**Comment by michaeldcain** [comment]
I think we've already had this discussion in Slack, but it's worth making it public :).

I'd recommend consolidating the list of statuses and using the expanded view to convey some of the details, since removing color will remove the number of possible variations.

What about:

- "normal/online/no notifs": solid bell
- "notifications" (seen or unseen message): solid bell with a dot
- "not working" (whether error or offline): outline of the bell, using the error message inside to differentiate

I think you had a good point about the "notifications" consolidation. Mind re-hashing it here?

**Comment by sirbrillig** [comment]
Good point, yes. I personally would like to have two different notification icons for unseen and unread because of the following situation:

1. I get three new notifications. The icon alerts me.
2. I click on the icon to see the notifications, but none of them is pressing and I decide I'll take a look a them later.
3. I get a fourth notification. 

If unseen and unread states were combined, there'd be no way for me to know that I had just received a new notification in in step 3 above since it would look the same. We _could_ combine unread with the normal state to solve this, but at least for me that would make me forget I had any notifications at all.
